server/src/repositories/media.repository.ts

```diff
@@ -141,6 +141,7 @@ export class MediaRepository {
       failOn: options.processInvalidImages ? 'none' : 'error',
       limitInputPixels: false,
       raw: options.raw,
+      unlimited: true,
```

Comment:
This should presumably be guarded behind some sort of configuration option (e.g. an environment variable).

Enabling this should be carefully considered in deployments where trust in users is weaker.

Reply:
Can you elaborate on the potential issues or security concerns?

Reply:
> You could presumably create a malicious image which consumes enough memory to crash the server

Thank you, this is what I meant